src/MahApps.Metro/MahApps.Metro.Shared/Controls/Dialogs/DialogManager.cs

```diff
@@ -428,7 +424,7 @@ public static Task HideMetroDialogAsync(this MetroWindow window, BaseMetroDialog
                 {
                     window.RemoveDialog(dialog);
 
-                    settings = settings ?? (dialog.DialogSettings ?? window.MetroDialogOptions);
+                    settings = settings ?? dialog.DialogSettings;
```

Comment:
`dialog.DialogSettings` is never `null`, therefore `window.MetroDialogOptions` is never evaluated